spec/factories/course_assessment_assessments.rb

```diff
@@ -13,6 +13,10 @@
       start_at { 1.day.from_now }
     end
 
+    trait :without_end_time do
```

Comment:
And I think it's better to give traits a with, vs a without. Do we have many negation traits?